error running "make create cluster"

/kind bug

root@capd-2:~/cluster-api-provider-gcp# make create-cluster
kind create cluster --name=clusterapi
Creating cluster "clusterapi" ...
 ✓ Ensuring node image (kindest/node:v1.15.3) 🖼
 ✓ Preparing nodes 📦
 ✓ Creating kubeadm config 📜
 ✓ Starting control-plane 🕹️
 ✓ Installing CNI 🔌
 ✓ Installing StorageClass 💾
Cluster creation complete. You can now use the cluster with:

export KUBECONFIG="$(kind get kubeconfig-path --name="clusterapi")"
kubectl cluster-info
# Apply provider-components.
root@capd-2:~/cluster-api-provider-gcp# make create-cluster
kind create cluster --name=clusterapi
Creating cluster "clusterapi" ...
 ✓ Ensuring node image (kindest/node:v1.15.3) 🖼
 ✓ Preparing nodes 📦
 ✓ Creating kubeadm config 📜
 ✓ Starting control-plane 🕹️
 ✓ Installing CNI 🔌
 ✓ Installing StorageClass 💾
Cluster creation complete. You can now use the cluster with:

export KUBECONFIG="$(kind get kubeconfig-path --name="clusterapi")"
kubectl cluster-info
# Apply provider-components.
kubectl \
        --kubeconfig=$(kind get kubeconfig-path --name="clusterapi") \
        create -f examples/_out/provider-components.yaml
namespace/cabpk-system created
namespace/capg-system created
namespace/capi-system created
customresourcedefinition.apiextensions.k8s.io/clusters.cluster.x-k8s.io created
customresourcedefinition.apiextensions.k8s.io/gcpclusters.infrastructure.cluster.x-k8s.io created
customresourcedefinition.apiextensions.k8s.io/gcpmachines.infrastructure.cluster.x-k8s.io created
customresourcedefinition.apiextensions.k8s.io/gcpmachinetemplates.infrastructure.cluster.x-k8s.io created
customresourcedefinition.apiextensions.k8s.io/kubeadmconfigs.bootstrap.cluster.x-k8s.io created
customresourcedefinition.apiextensions.k8s.io/kubeadmconfigtemplates.bootstrap.cluster.x-k8s.io created
customresourcedefinition.apiextensions.k8s.io/machinedeployments.cluster.x-k8s.io created
customresourcedefinition.apiextensions.k8s.io/machines.cluster.x-k8s.io created
customresourcedefinition.apiextensions.k8s.io/machinesets.cluster.x-k8s.io created
role.rbac.authorization.k8s.io/cabpk-leader-election-role created
role.rbac.authorization.k8s.io/capg-leader-election-role created
role.rbac.authorization.k8s.io/capi-leader-election-role created
clusterrole.rbac.authorization.k8s.io/cabpk-manager-role created
clusterrole.rbac.authorization.k8s.io/cabpk-proxy-role created
clusterrole.rbac.authorization.k8s.io/capg-manager-role created
clusterrole.rbac.authorization.k8s.io/capg-proxy-role created
clusterrole.rbac.authorization.k8s.io/capi-manager-role created
rolebinding.rbac.authorization.k8s.io/cabpk-leader-election-rolebinding created
rolebinding.rbac.authorization.k8s.io/capg-leader-election-rolebinding created
rolebinding.rbac.authorization.k8s.io/capi-leader-election-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/cabpk-manager-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/cabpk-proxy-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/capg-manager-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/capg-proxy-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/capi-manager-rolebinding created
secret/capg-manager-bootstrap-credentials created
service/cabpk-controller-manager-metrics-service created
service/capg-controller-manager-metrics-service created
deployment.apps/cabpk-controller-manager created
deployment.apps/capg-controller-manager created
deployment.apps/capi-controller-manager created
# Create Cluster.
kubectl \
        --kubeconfig=$(kind get kubeconfig-path --name="clusterapi") \
        create -f examples/_out/cluster.yaml
cluster.cluster.x-k8s.io/test1 created
gcpcluster.infrastructure.cluster.x-k8s.io/test1 created
# Create control plane machine.
kubectl \
        --kubeconfig=$(kind get kubeconfig-path --name="clusterapi") \
        create -f examples/_out/controlplane.yaml
kubeadmconfig.bootstrap.cluster.x-k8s.io/test1-controlplane-0 created
kubeadmconfig.bootstrap.cluster.x-k8s.io/test1-controlplane-1 created
kubeadmconfig.bootstrap.cluster.x-k8s.io/test1-controlplane-2 created
machine.cluster.x-k8s.io/test1-controlplane-0 created
machine.cluster.x-k8s.io/test1-controlplane-1 created
machine.cluster.x-k8s.io/test1-controlplane-2 created
gcpmachine.infrastructure.cluster.x-k8s.io/test1-controlplane-0 created
gcpmachine.infrastructure.cluster.x-k8s.io/test1-controlplane-1 created
gcpmachine.infrastructure.cluster.x-k8s.io/test1-controlplane-2 created
# Get KubeConfig using clusterctl.
bin/clusterctl \
        alpha phases get-kubeconfig -v=3 \
        --kubeconfig=$(kind get kubeconfig-path --name="clusterapi") \
        --namespace=default \
        --cluster-name=test1
I0912 01:04:20.704681     789 getkubeconfig.go:40] Getting target cluster kubeconfig.
I0912 01:04:20.704760     789 getkubeconfig.go:65] Waiting for kubeconfig from Secret "test1-kubeconfig" in namespace "default" to become available...
I0912 01:04:30.708354     789 getkubeconfig.go:65] Waiting for kubeconfig from Secret "test1-kubeconfig" in namespace "default" to become available...
I0912 01:04:40.708401     789 getkubeconfig.go:65] Waiting for kubeconfig from Secret "test1-kubeconfig" in namespace "default" to become available...
I0912 01:04:50.708360     789 getkubeconfig.go:65] Waiting for kubeconfig from Secret "test1-kubeconfig" in namespace "default" to become available...
I0912 01:05:00.708373     789 getkubeconfig.go:65] Waiting for kubeconfig from Secret "test1-kubeconfig" in namespace "default" to become available...
I0912 01:05:10.708350     789 getkubeconfig.go:65] Waiting for kubeconfig from Secret "test1-kubeconfig" in namespace "default" to become available...
I0912 01:05:20.708428     789 getkubeconfig.go:65] Waiting for kubeconfig from Secret "test1-kubeconfig" in namespace "default" to become available...
I0912 01:05:30.708362     789 getkubeconfig.go:65] Waiting for kubeconfig from Secret "test1-kubeconfig" in namespace "default" to become available...
I0912 01:05:40.708438     789 getkubeconfig.go:65] Waiting for kubeconfig from Secret "test1-kubeconfig" in namespace "default" to become available...
I0912 01:05:50.708385     789 getkubeconfig.go:65] Waiting for kubeconfig from Secret "test1-kubeconfig" in namespace "default" to become available...
I0912 01:06:00.708375     789 getkubeconfig.go:65] Waiting for kubeconfig from Secret "test1-kubeconfig" in namespace "default" to become available...
# Apply addons on the target cluster, waiting for the control-plane to become available.
bin/clusterctl \
        alpha phases apply-addons -v=3 \
        --kubeconfig=./kubeconfig \
        -a examples/addons.yaml
I0912 01:06:00.787402    3368 clusterclient.go:237] Waiting to acquire client...
F0912 01:06:00.787708    3368 alpha_phase_apply_addons.go:50] unable to create cluster client: failed to acquire new client: Get https://34.102.233.46:443/api?timeout=32s: EOF
Makefile:262: recipe for target 'create-cluster' failed
make: *** [create-cluster] Error 1

**What steps did you take and what happened:**
[A clear and concise description of what the bug is.]


**What did you expect to happen:**


**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]


**Environment:**

- Cluster-api version:
- Minikube/KIND version:
- Kubernetes version: (use `kubectl version`):
- OS (e.g. from `/etc/os-release`):

--- a/controllers/gcpmachine_controller.go
+++ b/controllers/gcpmachine_controller.go
@@ -365,6 +365,12 @@ func (r *GCPMachineReconciler) reconcileLBAttachment(machineScope *scope.Machine
 		return errors.Wrapf(err, "failed to add instance to group")
 	}
 
+	computeSvc := compute.NewService(clusterScope)
+	if err := computeSvc.ReconcileLoadbalancers(); err != nil {
+		gcpCluster := clusterScope.GCPCluster
+		return errors.Wrapf(err, "failed to reconcile load balancers for GCPCluster %s/%s", gcpCluster.Namespace, gcpCluster.Name)
+	}
+
 	return nil
 }
 

